volume/volume_test.go

```diff
@@ -1,11 +1,46 @@
+// +build !volumecreate
```

Comment:
Using a negative tag here makes the test command confusing. You run `go test -tags volumecreate`, which would cause one to assume that volume create tests would be run, but instead volume create tests aren't run with this.

Why not jut use `// +build volumecreate`? This way a plaing `go test` wouldn't run this, but `go test -tags volumecreate` would run this tests.